lib/rules/no-v-html.js

```diff
@@ -14,16 +15,38 @@ module.exports = {
       url: 'https://eslint.vuejs.org/rules/no-v-html.html'
     },
     fixable: null,
-    schema: [],
+    schema: [
+      {
+        type: 'object',
+        properties: {
+          ignorePattern: {
+            type: 'string'
+          }
+        },
+        additionalProperties: false
+      }
+    ],
     messages: {
       unexpected: "'v-html' directive can lead to XSS attack."
     }
   },
   /** @param {RuleContext} context */
   create(context) {
+    const options = context.options[0]
+    const ignoredVarMatcher = options?.ignorePattern
+      ? toRegExp(options.ignorePattern, { remove: 'g' })
```

Comment:
> Huh, apparently `vue/no-unused-vars` just uses `new RegExp(ignorePattern, 'u')` rather than the `toRegExp` helper. (I found the `toRegExp` helper in some other rule, don’t remember which one.)

I think `new RegExp(ignorePattern, 'u')` is indeed more suitable here. `toRegExp` is used in multiple rules that have an `ignore` option that allows both strings (that are compared as is) and strings containing patterns (e.g. `"/foo/"`).

Here, it is intentionally only one string option called "ignore_**Pattern**_", so the leading and trailing slashes are not necessary I think.

Reply:
Makes sense, will change.